docs/development.md

```diff
@@ -142,9 +181,24 @@ Sometimes, Eclipse does not show these options immediately after you set up the
 
 New developers should run `All tests` and have all of them passing at least once on their local environments.
 
-When running the test cases, if a few cases fail (this can happen due to timing issues), run the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse until they pass.
+When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse until they pass.
+
+### Running the test suite with IntelliJ
+
+Tests can be run using the configurations available under `Run → Play`. Several configurations are provided by default:
+* `All tests` - Runs `CI tests` and `Local tests`.
+* `CI tests` - Runs `src/test/testng-ci.xml`, all the tests that are run by CI (Travis/AppVeyor).
+* `Local tests` - Runs `src/test/testng-local.xml`, all the tests that need to be run locally by developers. `Dev green` means passing all the tests in this configuration.
+* `Failed tests` - Runs `test-output/testng-failed.xml`, which is generated if a test run results in some failures. This will run only the failed tests.
+
+Additionally, configurations that run the tests with `GodMode` turned on are also provided.
+More info on this can be found [here](godmode.md).
+
+New developers should run `All tests` and have all of them passing at least once on their local environments.
+
+When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab in IntelliJ until they pass.
 
-### Running the test suite outside Eclipse
+### Running the test suite outside Eclipse and IntelliJ
 
 Typically, we run the test suite within Eclipse, but core developers may prefer to run it outside Eclipse so that they can continue to use Eclipse while the test suite is running. If you wish to do such, given below is the procedure:
```

Comment:
within Eclipse -> within an IDE
continue to use Eclipse -> continue to use the IDE